icons/aspect-ratio-16x9.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect x="2" y="6.44" width="20" height="11.25" rx="2"/>

```

Comment:
This looks really ugly on low dpi, while it's not a true 16:9 aspect ratio, I suggest to go with the one below, and just call it `rectangle-horizontal`.
```suggestion
  <rect x="2" y="6" width="20" height="12" rx="2"/>
```

![image](https://user-images.githubusercontent.com/17746067/179494728-f5803d1d-2f8a-46c4-9109-cfd57eddbc31.png)

Reply:
Yes, you're right. I change the pixel sizes and also add a vertical version of the icon but it seems there are problems.

Reply:
Upon further consideration, while I think such rectangle icons would be useful, especially as building blocks of future icons, I'd prefer to keep them free of any explicit aspect ratio, such as: 
![image](https://user-images.githubusercontent.com/17746067/179700584-6bd07365-e083-4213-b4bd-8007dd2ae73c.png)